src/pipeline/rackhd/source_code/UnitTest.groovy

```diff
@@ -0,0 +1,78 @@
+package pipeline.rackhd.source_code
+
+def testWithSudo(String repo_name, String repo_dir){
+    try{
+        withCredentials([
+            usernamePassword(credentialsId: 'ff7ab8d2-e678-41ef-a46b-dd0e780030e1',
+                passwordVariable: 'SUDO_PASSWORD',
+                usernameVariable: 'SUDO_USER')
+        ]){
+            sh """#!/bin/bash -ex
+            trap 'echo $SUDO_PASSWORD |sudo -S chown -R $USER:$USER $repo_dir' SIGINT SIGTERM SIGKILL EXIT
```

Comment:
is it possible to be more specific, which folder under $repo_dir should be changed back to USER:USER?